easybuild/easyblocks/d/dm_reverb.py

```diff
@@ -94,7 +94,8 @@ def build_step(self, *args, **kwargs):
         # print full compilation commands
         bazel_build_opts += " --subcommands"
 
-        bazel_cmd = "bazel %s build %s %s" % (bazel_opts, bazel_build_opts, bazel_build_pkg)
+        bazel_cmd = "%s bazel %s build %s %s" % (self.cfg['prebuildopts'], bazel_opts, bazel_build_opts,
```

Comment:
This is a fix I needed to allow me to get around https://github.com/EESSI/software-layer/issues/192 which affects this package.